src/modules/fw_att_control/FixedwingAttitudeControl.hpp

```diff
@@ -231,6 +235,7 @@ class FixedwingAttitudeControl final : public ModuleBase<FixedwingAttitudeContro
 	ECL_WheelController		_wheel_ctrl;
 
 	void control_flaps(const float dt);
+	void control_spoilers(const float dt);
```

Comment:
I know I'm being annoying - but while we're adding new methods, can we incrementally start adding the `@brief` and `@param[in]` description + units here? `dt` could mean many things to someone new to the code.

Reply:
https://github.com/PX4/PX4-Autopilot/pull/19329/commits/98ea2aeb1c80f716c5335c194fbe02116df85ca6